docs/how-to/uc-boot-parameters.rst

```diff
@@ -0,0 +1,70 @@
+How to Configure Boot parameters on Ubuntu Core 
+================================================
+
+Different from classic versions of Ubuntu (Desktop and Server), `Ubuntu Core`_ is a minimal, secure and strictly
+confined operating system designed to run on embedded systems, such as IoT devices.
+Ubuntu Core is a snap-based operating system, meaning it uses snap packages to manage applications and services.
+Its read-only nature makes it a very secure and reliable operating system but also makes it a bit more difficult to
+configure certain aspects of the system, such as boot parameters.
+
+
+Configure boot parameters
+-------------------------
+
+As UC doesn't have the file :code:`/etc/default/grub` mapped for the kernel parameters, neither the `update-grub`_
+utility, it's necessary to use the `snap set`_. There are two ways to set the kernel parameters:
+
+- `system system.kernel.cmdline-append`_
+- `system system.kernel.dangerous-cmdline-append`_
+
+The first one is used to dynamically append permitted kernel boot parameters that are verified against an `allow list`_
+in the `gadget snap`_, while the second one is used to append kernel boot parameters considered risky according to that
+specific gadget snap. When using the :code:`pc-gadget` or the :code:`pi-gadget`, it is advisable to use the 
+`system.kernel.dangerous-cmdline-append` option, as the allow list is not present in those gadget snaps. 
+However, in the case of `building a gadget snap`_ independently, it is possible to implement the allow list.
```

Comment:
The fixed char and justified text is making it hard to suggest on the right sentences.

```suggestion
specific gadget snap.

When using the trial images, you need to use the 
`system.kernel.dangerous-cmdline-append` option, as the allow list is not present in the reference gadget snaps. 

When creating a custom `gadget snap`_ for a real-time Ubuntu Core system, it is advisable to add an allow list with the relevant parameters.
```

I think maybe we shouldn't even mention `system system.kernel.cmdline-append` and so avoid having to explain it. 

These and the custom gadget below to another how-to.

Reply:
We could omit this, but this is a small detail to explain (two sentences), I don't see much benefit on having this information missing. 
We would already have to explain what this command does anyway, if we only explain the `dangerous` version, it would give the wrong idea that it is the only way to do it, so it would be necessary to at least refer it anyway.